src/index.css

```diff
@@ -187,4 +187,4 @@ hr {
   border: none;
   background-color: #0290ff;
   color: #fefefe;
```

Comment:
Hello @SirriRyisa, You have done well in this project. I am able to add and remove books to the project as per the requirements! However, the styles in the project do not load and this is because the `index.css` file is not rendered well in the `index.js` as you can see below.

- [ ] I think it should be rendered this way 👉🏻 `import './index.css'`  instead in the `index.js`.

![image](https://user-images.githubusercontent.com/4247740/157887659-85e61ac9-3cab-4f27-92c8-67cdc646b0ac.png)

Reply:
Hi Cathela, thank you for the review. I am not sure why it is not reading on your path but I have a working version here